backend/src/mongo/models/Session.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from 'mongoose';
+
+const sessionSchema = mongoose.Schema({
+  isFinished: {
+    type: Boolean,
+    default: false,
```

Comment:
Again I don't think this is complete

Reply:
I have removed the Session model, so one of you guys can implement it.